package.json

```diff
@@ -13,7 +13,7 @@
   ],
   "repository": "googleapis/nodejs-datastore-session",
   "peerDependencies": {
-    "@google-cloud/datastore": "^4.0.0"
+    "@google-cloud/datastore": "^5.0.0"
```

Comment:
@bcoe @JustinBeckwith should we make this ^4?

Reply:
I don't think it's a big deal if we ask folks to be on the latest `datastore`, but I think we should call this a breaking change potentially? since it will force folks to update their peer dependency to 5.x.